Maquetado/scss/bookDetail.scss

```diff
@@ -2,88 +2,117 @@
 @import 'mixins';
 
 .bookDetailBackground {
-    align-items: center;
-    background-color: $seashell;
+  align-items: center;
+  background-color: $seashell;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  padding-top: 25px;
+  padding-bottom: 28%;
+
+
+  .backButtonContainer {
     display: flex;
-    flex-direction: column;
-    gap: 15px;
-    padding-top: 25px;
-    padding-bottom: 28%;
+    flex-direction: row-reverse;
+    position: relative;
+    right: 38%;
 
+    .backButton {
+      font-size: 18px;
+    }
 
-    .backButtonContainer {
-      display: flex;
-      flex-direction: row-reverse;
-      margin-right: 80%;
+    .backArrowImage {
+      margin-right: 10px;
+    }
+  }
 
-      .backButton {
-        font-size: 18px;
-      }
+  .bookContainer {
+    display: flex;
+    background-color: $white;
+    box-shadow: 4px 4px 9px -4px rgba($black, 1);
+    margin-top: 20px;
+    padding: 30px;
+    width: 65%;
 
-      .backArrowImage {
-        margin-right: 10px;
-      }
+    .bookImage {
+      height: 385px;
+      position: relative;
+      width: 250px;
+      margin-right: 30px;
     }
 
-    .bookContainer {
-      display: flex;
-      background-color: $white;
-      box-shadow: 4px 4px 9px -4px rgba($black, 1);
-      width: 65%;
+    &::after {
+      content: '';
+      background: url('./resources/img/badge.png');
+      background-size: cover;
+      height: 80px;
+      left: 33.5%;
```

Comment:
I'm not saying that this achieves the expected result, but generally when we use a percentage value like xx.x% it means that there's something in our elements width / positioning that can be improved